lib/bonny/config.ex

```diff
@@ -3,6 +3,13 @@ defmodule Bonny.Config do
   Operator configuration interface
   """
 
+  @project_name Mix.Project.config()
```

Comment:
@akasprzok This always ends up being "bonny", right? It does not take the project config of the project using bonny...